spec/api/default_api_spec.rb

```diff
@@ -0,0 +1,637 @@
+=begin
```

Comment:
i'm not going to check them in since they don't do anything. we can add them back if we want to later